tests/modules/services/window-managers/i3/i3-fonts-expected.conf

```diff
@@ -104,3 +104,4 @@ bar {
 
 
 
+
```

Comment:
Actually, we have several useless newlines here at the end, so if anything, we should be removing them.

```suggestion
```